From 7b56e1341c486422973dfa890158872200527b51 Mon Sep 17 00:00:00 2001
From: Hood Chatham <roberthoodchatham@gmail.com>
Date: Sat, 25 Dec 2021 20:31:40 -0800
Subject: [PATCH] Remove cuncsd, dorcsd, sorcsd, zuncsd

These functions don't work because their definitions involve code that isn't 
fortran 77 compliant. In order to get _flapack.so to link correctly, we need
to provide stub definitions for these functions, but they don't work. Remove
them to make sure people get an `AttributeError` if they try to use them 
rather than just having weird buggy code.

---
 scipy/linalg/__init__.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/scipy/linalg/__init__.py b/scipy/linalg/__init__.py
index e88479b48..746bf3dce 100644
--- a/scipy/linalg/__init__.py
+++ b/scipy/linalg/__init__.py
@@ -191,6 +191,12 @@ Low-level routines
    `scipy.linalg.cython_lapack` -- Low-level LAPACK functions for Cython
 
 """  # noqa: E501
+from . import lapack
+del lapack.cuncsd
+del lapack.dorcsd
+del lapack.sorcsd
+del lapack.zuncsd
+del lapack
 
 from .misc import *
 from .basic import *
-- 
2.25.1

